grails-app/assets/javascripts/spApp/service/biocacheService.js

```diff
@@ -889,7 +889,7 @@
                                 return {
                                     q: $.merge([q], fq),
                                     wkt: wkt,
-                                    qid: "qid:" + data.qid,
+                                    qid: data.qid, //qid has been reformated to "qid:xxxxxx"
```

Comment:
I thought this was fixed in hotfix - won't this result in a git conflict now?

Reply:
I merged back from hotfix